sys-apps/pacman/pacman-5.0.2.ebuild

```diff
@@ -0,0 +1,84 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="6"
+
+PYTHON_COMPAT=( python2_7 )
+
+DESCRIPTION="Archlinux's binary package manager"
+HOMEPAGE="https://archlinux.org/pacman/"
+
+if [[ ${PV} == "9999" ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://git.archlinux.org/pacman.git"
+	EGIT_BRANCH="master"
+else
+	SRC_URI="https://sources.archlinux.org/other/pacman/${P}.tar.gz"
+	# Do *not* re-add ~x86!
+	# https://www.archlinux.org/news/phasing-out-i686-support/
+	KEYWORDS="~amd64"
+fi
+
+LICENSE="GPL-2"
+SLOT="0"
+
+IUSE="curl debug doc +gpg"
+
+COMMON_DEPEND="app-arch/libarchive[lzma]
+	gpg? ( >=app-crypt/gpgme-1.4.0 )
+	dev-libs/openssl:=
+	curl? ( net-misc/curl )
+	sys-devel/gettext
```

Comment:
gettext required at runtime looks suspicious. If it's really needed a comment in the build would help.